training/src/main/java/wolox/training/models/Book.java

```diff
@@ -40,6 +45,9 @@
     @Column(nullable = false)
     private String isbn;
 
+    @ManyToMany(mappedBy = "books")
+    private List<User> users = new LinkedList<>();
```

Comment:
What do you think about using sets instead of lists? Sets dont allow duplicated objects and have better performance, you can see more about this [here](https://vladmihalcea.com/the-best-way-to-use-the-manytomany-annotation-with-jpa-and-hibernate/).